prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml

```diff
@@ -1343,6 +1343,101 @@ periodics:
       - 'E2E_ARGS=--stress -run=TestStress*'
       - 'GCP_CLUSTER=stress-test'
 
+# Experimental prowjob to verify the new create-clusters flow on standard.
+# If stable, this is intended to replace the above testgroup-based definitions
+- name: standard-regular
+  interval: 1h
+  annotations:
+    testgrid-dashboards: googleoss-kpt-config-sync-misc
+    testgrid-tab-name: standard-regular
+  cluster: build-kpt-config-sync
+  decorate: true
+  decoration_config:
+    timeout: 2h
```

Comment:
will it start another one if the previous is still running?

Reply:
My understanding is that periodic jobs won't run concurrently, i.e. it will wait for the previous to complete. The docs seem to confirm this
```
	// Consecutive jobs are run at `interval` duration apart, provided the
	// previous job has completed.
```